### PR TITLE
68 explore page

### PR DESCRIPTION
**Pull request by BBruington**
Hey came across with a few issues but here's what i have thus far: 
Screenshot 2024-02-01 at 6 56 47 PM

Here's how it looks when you click on an image:
Screenshot 2024-02-01 at 7 01 26 PM

here's how it looks on a smaller screen: 
Screenshot 2024-02-01 at 6 53 58 PM

It seems that the component is designed to be a the length of a screen. Here's what happened when i messed around to increase the images: 
Screenshot 2024-02-01 at 6 52 00 PM
Clicking on the image made it go to the top of the screen and i think i would have to manipulate the atom to fix it unless i'm missing something.

### CODE REVIEW

File: apps/chameleon/utils/cn.ts
Comment (Balance8) [inline apps/chameleon/utils/cn.ts:1]: we dont need this as we already have one

File: apps/chameleon/app/api/send/route.ts
Comment (Balance8) [inline apps/chameleon/app/api/send/route.ts:1]: we deleted this in @JakeJeongAiden pr no need to do it here, once he merges we can rebase and it'll go away

**Review by Balance8 — CHANGES_REQUESTED** [review]
Do you want to add to the ticket to figure out how to extend the component so that it will scroll and load more so that it doesn't break, as you mentioned?

**Comment by BBruington** [comment]
Here's how it looks now when you click on a photo while scrolled to the bottom of the screen:
Screenshot 2024-02-02 at 4 03 54 PM

**Comment by BBruington** [comment]
Also here's how it looks on a small screen when you click on a photo at the very bottom: 
Screenshot 2024-02-02 at 4 08 07 PM

File: apps/chameleon/components/ui/layout-grid.tsx
Comment (Balance8) [inline apps/chameleon/components/ui/layout-grid.tsx:1]: I think we have an original file already of this

Comment (BBruington) [reply apps/chameleon/components/ui/layout-grid.tsx:1]: ight i just changed the import to the one in aceternity and deleted mine

**Review by Balance8 — APPROVED** [review]
[crabrave](https://app.graphite.dev/macros/Ionia-Devs)

**Comment by JakeJeongAiden** [comment]
Looks good!

**Review by DinDin985 — APPROVED** [review]
lgtm! 👍

**Review by JakeJeongAiden — APPROVED** [review]
Looks good

**Pull request merged**